### DISCUSSION TITLE
Resurrecting $BICO Tokenomics

### DISCUSSION THREAD

**Discussion by woogieboogie-jl** [bcnmy/biconomy-client-sdk#584] [Ideas]
# Ideation - Revamping $BICO Tokenomics

**Written by Jaewook (tg: @woogieboogie_jl), reviewed and commented by Romain (tg: @ReezaMe) and David (tg: @couchecouche)**

_:This proposal aims to provide the project with a proper heads-up from community members and to establish a dedicated communication channel for token holders to contribute to shaping the tokenomics. The ideas presented here are not intended to represent the final form of any effective tokenomics changes._

## Table of Contents

1. Introduction: $BICO needs a revamp
2. The Revamp
  2-1. Possible Deflationary Modifications on the $BICO Tokenomics 
  2-2. Implementation Example: The $BICO Bucket Auction
3. Further Considerations: “What game are we playing?”
4. Conclusion: Call to Action

---

## 1. Introduction

The current state of $BICO tokenomics strongly presents an opportunity for improvement. While Biconomy has explored several token-related initiatives—such as Biconomy staking via $BICO and $BBPT, the use of $BICO in the Hyphen bridge for gasless transactions, and Biconomy Loyalty airdrops—these efforts have yet to deliver sustainable results, particularly in terms of consistent community engagement and price stability of the $BICO token.

Recently, Biconomy has experienced challenges related to token value and market capitalization. Over the past year, while the altcoin market has grown by nearly 95%, $BICO’s price has seen a modest increase of 9%. In the last six months, $BICO’s token price has decreased by 68%, and its market cap has declined by 59.4%, compared to a 32.3% decline in the broader altcoin market (according to CoinGecko’s altcoin index). This underperformance suggests that a renewed focus on developing comprehensive tokenomics could be beneficial for Biconomy's long-term success.

Importantly, I had a conversation with Aniket, one of Biconomy's co-founders, during the KBW(Korea Blockchain Event), and he agreed that this initiative may be community-driven. This alignment with the team’s vision provides a rare opportunity for a collaborative, innovative approach to Biconomy’s future.

---

## 2. The Revamp

Historically, tokens with properly aligned incentivization mechanisms have proven critical to a project's growth and success. Take, for example, the Curve Wars, where token-based incentives drove unprecedented competition and adoption between LP protocols. Similarly, SushiSwap briefly overtook Uniswap, and Blur managed to capture significant market share from OpenSea, largely due to their effective use of token incentives.

We believe Biconomy, along with proper $BICO tokenomics, can achieve greater success through a carefully planned tokenomics redesign. Biconomy already has strong technical infrastructure and product potential revolving around the ERC-4337 backed Account Abstraction toolkits, and a revamped tokenomics strategy—referred to here as the "Biconomics Revamp"—has the potential to capitalize on that foundation.

### 2-1. Possible Deflationary Modifications on the $BICO Tokenomics

At the current state, we can think of $BICO tokens’ utility within the product ecosystem, ranging from Biconomy SDK, Bundlers, Relayers, Paymaster and others. While the example burning mechanism is well illustrated in section 2-2: “Implementation Example: The $BICO Bucket Auction,” this section points out on which product and at what part of its function we can implement deflationary features for the token.

#### 1. Bundling Fees with Burn Component

While Bundlers in Biconomy’s AA infrastructure currently handles only one UserOp for one bundler, after bundler is able to aggregate multiple UserOperations into one Ethereum transaction, we believe there's room for fees to be made. From there we can introduce a burn mechanism by requiring part of the fees earned by Bundlers to burn $BICO.

- For every UserOperation bundled:
    - A portion of the fees (e.g., 1-2%) is paid in $BICO.
    - 50% of that portion is burned, reducing the supply of $BICO permanently.
    - The remaining portion is retained by the Bundler for incentivization.

This could be implemented via smart contracts that automatically burn $BICO whenever a UserOperation fee is processed, adding transparency and decentralization to the burn process.

#### 2. Paymaster Fee with Burn Component

As far as I understand, Biconomy offers Paymaster service, which are utilized in two ways: (1) dApps use it to sponsor gas for users, and (2) Users pay gas in any token. For each case, we can think of how burn component can be implemented:

- The fee could be paid in $BICO by dAppss that want to utilize paymasters for sponsorship purposes. And for individual users, a certain premium is collected with accepted tokens. 
- Tokens collected from individual users can be swapped back to $BICO. Then along with fees collected from both (1) and (2), a percentage of the collected $BICO fee can be burned. 
- Dynamic burn rates could be introduced based on the volume of transactions or network conditions. For instance, if network congestion is low, a higher percentage of $BICO could be burned (e.g., 2%), while during peak times, a lower percentage could be burned (e.g., 1%) to maintain service stability.

#### 3. Smart Wallet Deployment Fee with Burn

As AA enables smart contract wallets with advanced features like social recovery or multisig, Biconomy could introduce wallet deployment fees to be optionally paid by $BICO.

- Whenever a user or DApp deploys a new smart contract wallet, they pay a small deployment fee in $BICO.
- A portion of this deployment fee is burned (e.g., 10-20%).

This could be an attractive burn mechanism because wallet deployment would increase as more users adopt AA features like social recovery and multisig wallets, ensuring a steady and growing burn over time.

#### 4. BICO Staking with Burn for Governance or Fee Reduction

Biconomy could introduce a discount on its AA services to $BICO staking users, which would incentivize both $BICO usage and user engagement on its products.

- For example:
    - Stakers could receive a fee reduction or priority in transaction processing, sometimes with different perks via different staking levels.
    - A small portion of the staked $BICO tokens could be burned over time as part of the staking rewards distribution mechanism.
    - Stakers with over a certain level of $BICO staked are eligible to make a ‘prioritized’ transaction or sometimes even at a free cost with a different message sender.

This encourages long-term holding and staking while also introducing periodic burns.

#### 6. Governance and Community Proposals

To align $BICO token holders with Biconomy’s long-term vision, governance decisions or community proposals could involve a burn mechanism and its parameter changes.

- All the procedures and parameters listed for the $BICO burn may be modified by a governance vote, ensuring that governance engagement actively contributes to controlling $BICO’s tokenomics and governance.
- In addition, even casual proposals that impact Biconomy and its users can implement a governance proposal-accept feature that operates via governance power distributed pro-rata to the amount of $BICO tokens staked at the staking portal. Enabling $BICO’s governance utility upon approving a new feature, a project, etc.

These approaches would make $BICO essential to the infrastructure’s functioning and provide clear utility for holding the token, ensuring a valid value accrual pipeline of the $BICO token that aligns with Biconomy’s product scheme.

---

### 2-2. Implementation Example: The $BICO Bucket Auction

To foster sustained demand for the $BICO token while aligning with Biconomy’s ecosystem growth, we propose a periodic auction mechanism called the $BICO Bucket Auction, which was heavily inspired by Injective’s $INJ Auction. This system plans to introduce an indirect use of the $BICO token, promoting buying pressure, value accrual, and community engagement without forcing users to hold $BICO for their daily activities. Below is a detailed breakdown of the mechanism:

#### 1. Implementation Overview

- **Fees collected**: From above-listed product pipelines Biconomy provides.
- **Dedicated auction platform**: A dedicated auction page will be created to host and display the auctions in real-time. This platform will allow users to view the current size and composition of the basket, track ongoing auctions, and place bids.

**Biconomy SDK**: SDK users will continue to transact in their preferred tokens, such as USDC, USDT, MATIC, ETH, and other supported tokens. This ensures they aren’t required to hold $BICO for everyday use, maintaining flexibility and ease of participation in Biconomy’s ecosystem.

**Fee Allocation for Auctions**:
- Transaction Fees: As part of the current process, UserOp fees will continue to be collected in the token of the user’s choice. However, a portion of these fees will be allocated to a special auction pool.
- Proportion of Fees (TBD): A specified percentage of these collected fees will be redirected into the auction pool, which will form a basket of tokens to be bid on.

#### 2. Auction Basket

**The Basket Appeals Increasing Value**: Over time, as Biconomy grows and transaction volumes increase, the auction basket becomes more appealing due to its larger size and diversification. This dynamic serves to encourage higher levels of bidding participation.

- **Diverse Token Pool**: The fees collected in various tokens (e.g., USDC, MATIC, ETH, etc.) will be pooled to form a "basket" of tokens.
- **Dynamic Growth**: The size and diversity of the basket will be determined by two key factors:
    - Volume of UserOps: The more UserOps (transactions) that occur, the larger the basket becomes, which will incentivize participation.
    - Blockchain Usage: As more blockchains are used in transactions, the basket will diversify further, potentially including more tokens from different networks.

#### 3. Auction Mechanism

- **Bidding in $BICO**: Multiple bidders will compete for the auction basket by bidding using $BICO tokens. The auction creates an indirect use case for $BICO, driving up demand in the market.
- **Scheduled Auctions**: The auctions will be held periodically (e.g., weekly or bi-weekly), with each period corresponding to the amount of collected fees. The frequency of auctions can be adjusted based on network activity and governance decisions.
- **Buying Pressure**: As multiple bidders compete for the basket, this bidding competition should create buying pressure on the $BICO token. The result is an increased demand for $BICO in the market, contributing to a healthier token ecosystem.
- **Market Impact**: The competition between bidders may drive the price of $BICO upwards, as participants acquire more tokens in preparation for each auction.

#### 4. Auction Outcome

- **Winning the Basket**: At the end of the auction, the highest bidder will win the basket of tokens. The winner will exchange the $BICO tokens they bid for the basket of various assets.
- **Potential Discount**: Depending on the competitive dynamics of each auction, the winner may gain the basket of tokens at a value higher than the $BICO they spent, effectively earning a “discount” through their participation.
- **Deflationary Mechanism**: Burning the Winning Bid: The $BICO tokens used by the winning bidder will be burned, permanently removing them from circulation. This consistent burning mechanism will help reduce the overall supply of $BICO over time, contributing to an increased scarcity of the token.

This auction mechanism introduces a valuable new utility for $BICO, rewarding long-term holders ("golden hands") by reducing the circulating supply as others participate in the auctions. It also expands Biconomy's reach by attracting new participants, such as bridge operators, who are interested in bidding on already distributed baskets of multi-chain assets. Ultimately, this model aligns with Biconomy's ecosystem growth, creating value for both committed holders and new entrants.

---

## 3. Further Considerations: “What Game Are We Playing Here?”

While deflationary tokenomics are often seen as a means of preserving and potentially increasing the value of a token for investors, it is crucial not to overlook the importance of the ecosystem from which the token accrues its value. A highly deflationary token model that discourages network activity—such as users sending transactions—can be counterproductive, as it reduces the very market activity that gives the token its value. This is why, in economic theory, inflation often plays a positive role by encouraging activity, rather than stifling it.

As we explore the "Biconomy Revamp," the long-term success of this initiative will depend heavily on how well the tokenomics are aligned with Biconomy’s product roadmap. Before proposing changes from the community side, it is essential to ensure that any modifications to tokenomics are in harmony with both Biconomy’s current strategy and its future direction.

### So, What Game Are We Playing?

Until late 2023, Biconomy has primarily operated as a blockchain infrastructure provider, akin to a SaaS model, focusing on supporting decentralized applications (dApps) by leveraging Account Abstraction (ERC-4337) technology. While AA is increasingly becoming a critical component of user applications, the competition within the AA infrastructure space is fierce. This raises an important question: Success in AA infrastructure may demand economies of scale and high-level networking capabilities, as evidenced by the recent surge of UserOps on Base, with Alchemy serving as the AA provider. Given this dynamic, will Biconomy continue to focus solely on infrastructure, or are there plans to shift strategy for larger-scale growth? Understanding this is key to developing the right tokenomics that can sustain Biconomy’s future.

In my view, one potential strategy for Biconomy could involve expanding beyond backend infrastructure to also develop consumer-facing applications. This diversified approach would enable Biconomy to tap into multiple revenue streams, increase competitiveness, and foster sustainable growth in the evolving market. However, the community would greatly benefit from a clearer understanding of Biconomy’s overall strategy before recommending specific tokenomics changes.

### Request for Strategic Clarity

Therefore, myself and other community members, including DT (@couchecouche) and Romain (@ReezaMe), would appreciate insights into the broader strategic direction Biconomy intends to pursue. Understanding the "game" we are playing in the AA infrastructure space will enable us to propose a more viable and synergistic tokenomics model that aligns with Biconomy’s roadmap and product goals.

For instance, if Biconomy plans to integrate consumer-facing applications alongside its backend infrastructure, it may be beneficial to explore alternative tokenomics models—such as a dual-token system. This approach could differentiate the tokenomics for backend services from those supporting Biconomy’s base AA stack, providing greater flexibility and aligning incentives with both user-facing and infrastructure-focused aspects of the business.

---

## 4. Conclusion: Call to Action

After reviewing this proposal, there are key action items that, as a community member, I expect the Biconomy team to consider and leave meaningful feedback.

1. **Feedback on Section 2: Implementation of Deflationary Tokenomics**:
   - Need feedback on both (1) where we can implement token burn / give utilities to $BICO, and (2) the $BICO burn-auction mechanism. Note that $BICO burn auction is a mere Injective-inspired example of how we can implement the burn mechanism within Biconomy’s existing product ecosystem.

2. **Evaluating Revenue and Token Supply**:
   - To execute a successful tokenomics revamp, we need insights into Biconomy's current revenue streams. Specifically, how much of the revenue from the bundler-paymaster system can be allocated towards token burning without jeopardizing the project runway? We’ve currently only taken (1) bundlers, (2) smart wallets, and (3) paymasters as possible sources of revenue for Biconomy, but if more revenue streams exist that can help reduce the number of $BICO tokens, it would be great to receive comments on this, as knowing all possible revenue streams will help design a sustainable model for supply reduction.

3. **Community-Based Tokenomics Revamp Initiative**:
   - To ensure transparency and community involvement, we recommend announcing a one-month, community-driven tokenomics revamp initiative. This could be done via an official statement and the creation of a dedicated Discord channel where community members can contribute their ideas and participate in discussions.

4. **Request & Open Discussion on Biconomy’s Strategic Plans**:
   - We’d like to understand Biconomy’s competitive landscape (strength/weakness/opportunity/threat), to get a better ‘grasp’ on what game we’re playing and come up with a tokenomics structure that works along with the plan.

By working together on this "Biconomics Revamp," we believe that Biconomy can reverse its current underperformance and unlock significant growth opportunities moving forward. With the right strategy, we believe that there's still enough room for $BICO to regain its competitive edge and ensure the long-term sustainability of the Biconomy ecosystem along with its builder-centric product scheme.

**Comment by woogieboogie-jl** [comment]
We'd love to hear feedback from the team, or even from other community members with other novel ideas and even in-depth methodologies on how we can implement a sustainable $BICO tokenomics!

**Comment by Anmol37** [comment]
The burning mechanism seems right ! 
But the community has been allocated 38% 
Where did they went ? To staking whole ?
Can you please reply accordingly as only 1000 users had gotten 2k Bico each during it's airdrop 

The unlock is continuing and if possible add some transparency about where they are being allocated and can we expect more from this allocation part back to community ?

**Reply by anibico** [reply]
That's right, we still have majorly in the community vault to be used further for community development

**Reply by woogieboogie-jl** [reply]
I think this is one of points that could be illustrated from the biconomy team. @anibico are you Aniket from Biconomy? or another community member walking by?

**Comment by anibico** [comment]
I truly appreciate your thoughtful efforts and initiative in drafting this open proposal aimed at improving our token model. As you highlighted, the $BICO tokenomics offer significant room for innovation and optimization, and your input comes at a critical juncture. The challenges we’ve faced in terms of token value and community engagement are on our radar, and the proposal you’re working on aligns with our commitment to building a more resilient and vibrant ecosystem.

Regarding the bucket auctions, i am still trying to foresee the true demand for these kinda plays. May be would be worth brainstorming with you the pros and cons there but i do see its a long terms game. 

On the strategic clarity, there are a few areas we are working towards. Next couple of weeks will be important for you guys to take a closer look on our socials. There are a bunch of announcements that will be going live

**Reply by woogieboogie-jl** [reply]
To avoid confusion - Injective type of burn auction can take few roadmaps to actually implement irl.

But it's a working model -[ did (1) paired t-tests, (2) multivariable ](https://github.com/woogieboogie-jl/biconomics-revamp)modelling and it does have longer-term effect on the tokenomics but shows no significance in short term. So it's a good working tokenomics - but should be looked at in terms of sustainability, not short pump-dump magic.

I think the discussion should first revolve around 'what we can put into the basket initially' rather than 'how we're going to burn the basket' since our purpose is to streamline biconomy's on-chain product revenue to holder benefits / token deflation.

Can chads from the Biconomy team leave feedbacks on if the above suggested points of Biconomy burn makes sense + aligns with the Biconomy's long-term product engagement and strategy?

**Comment by ApeinCave** [comment]
Wrote in tg and write it here. All that looks very interesting. I am for any movement, experiments are better than doing nothing for years. So far it looks pretty complicated for a normie. More interesting is how to get the mass user to use BICO.
 I had an idea of ​​a concept of postponed fee payment using BICO. For example, the user must stake BICO to use it. The transaction fee is not paid immediately, from the user's wallet, but during the period of interest payments on staking from the amount that the user receives. The donor application pays for the fee, which ultimately receives these payments in the BICO token from users interests payment. In fact, for the user, this is the absence of commissions in general, this is especially beneficial for high-frequency traders, he can "lose" if the purchase price of BICO was significantly higher than the current market price. The donor application is also interested in being an active market player in order to make a profit, since the difference in the price of fee at the time of registration of the fee payment by the user in $/BICO and at the time of app will get its payment, can be very profitable for the donor. If the user exceeds the limit of the amount, he can either pay the missing part from the staked amount, or pay in any other token of his\apps choice, or transfer to the next period.

**Reply by woogieboogie-jl** [reply]
Noice. think that's also a viable option on the execution side of things - delayed type of transaction prepaid with $BICO as collateral. After receiving feedback till this week. I'll try to summarize all the community + biconomy team's feedback and try to come up with a good idea that could make most of stakeholders here (not all) happy

**Reply by ApeinCave** [reply]
Yeah, go for it! ) In my thoughts we need mainly unique ideas, that nobody have, that will stimulate to use biconomy as an aa provider for masses, not just\only price influence ideas. And it will be hard, they'v lost most of the community bcs they focused only at b2b segment and did not show price action. My thoughts also was about presenting for bico stakers unique tx routes, especialy in time when blockchains r heavy loaded , but technicaly i dont have an idea how to realize it without opening validator nodes at every blockchain for that. The only thing that comes to my mind is, if the user, in addition to the cost of the transaction in the native currency of chain, will pay an additional commission in Bico, increasing the cost of the commission and getting priority in the queues. At the same time, stimulating validators that will get that bico bonus, to participate in active trading of the coin. Part of comission can be burned, to turn on deflationary model as you proposed. And again, i have no clue, if its technicaly possible.

**Reply by woogieboogie-jl** [reply]
I think your suggestion makes sense if Biconomy team has plans to transition it's AA infra to support multi-chain type of AA infra and also requires $BICO token to be utilized in the form of multichain token - like layerzero's OFT(Omni-chain Fungible Token), Axelar’s ITS, Hyperlane's Warp, xERC and many more standards to come. Good take.

**Reply by ApeinCave** [reply]
Any news?

**Reply by woogieboogie-jl** [reply]
well the good news is that i know that they are preparing for huge updates regarding tokenomics - expecting them to make responses on this forum after important updates been publicly delivered.

**Comment by woogieboogie-jl** [comment]
Since Biconomy team acquired Klaster protocol and is having their product lineup [expanded to somewhat chain-abstracted execution environment (MEE)](https://www.biconomy.io/post/modular-execution-environment-supertransactions), this discussion needs to be refined to fit into the newly changed context.

I'm still processing this but after giving few thoughts during this week and meeting builders from the team, I'll try to publish / update this proposal where tokenomics could be implemented better in the newly changed environement.

**Reply by woogieboogie-jl** [reply]
https://x.com/woogieboogie_jl/status/1860381133629981013

I've wrote a thread, and I've talked to Biconomy team personally in Devcon.

Lot of stuff for the team to decide, align before getting down to tokenomics, and it might take around 1M to get this settled. 
My conclusion is that with MEE stuff, tokenomics value capture should be somewhat easier and better (cause it's a network), and I'd like to discuss this further once the team gives a go on how things will unfold with MEE in place.

**Reply by ApeinCave** [reply]
Few years later.
